server/views/layout.hbs

```diff
@@ -51,6 +51,18 @@
     crossorigin="anonymous"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn"
     crossorigin="anonymous"></script>
+
+    <script>
```

Comment:
Add the `<script>` only to the `articles-list`. It is not needed in other pages so no need for it to be layout